spotbugs/etc/messages.xml

```diff
@@ -1655,7 +1653,7 @@ factory pattern to create these objects.</p>
     <Details>
 <![CDATA[
 <p> Checks for calls to methods which perform a byte to String (or String to byte) conversion using the user's default
- platform encoding. This can cause the application behaviour to vary between platforms.  </p>
+ platform encoding. This can cause the application behavior to vary between platforms.  </p>
```

Comment:
@KengoTODA Do we have an official policy on British vs US spelling?

Reply:
I think we do not have. If necessary, please raise an issue. :)